codegen/masm/src/emulator/mod.rs

```diff
@@ -1242,6 +1255,11 @@ impl Emulator {
         // control flow effect occurred to reach it
         let ix_with_op = state.next();
         if let Some(ix_with_op) = ix_with_op {
+            if self.print_trace {
+                eprintln!("stk: {}", self.stack.debug().pretty());
+                eprintln!("mem: {:?}", self.memory);
```

Comment:
**Suggestion:** Might be nicer to put memory first, followed by the operand stack and current op, since that orders them from least significant to most significant (and memory is likely to be very noisy in larger programs with rodata and such, which might push visibility of the operand stack off screen).